### PR TITLE
add 0.11.14 image

### PR DESCRIPTION
**Pull request by hoshitocat**

### CODE REVIEW

**Comment by RossyWhite** [comment]
:lgtm:

**Comment by hoshitocat** [comment]
thank you 🎉

File: 0.11.13/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM hashicorp/terraform:0.11.13
+FROM hashicorp/terraform:0.11.14
```
Comment (RossyWhite) [inline 0.11.13/Dockerfile]: why?

**Comment by RossyWhite** [comment]
LGTM, truly

File: 0.11.13/Dockerfile (hunk 1 above)
Comment (hoshitocat) [reply 0.11.13/Dockerfile]: Sorry 🙇

**Comment by hoshitocat** [comment]
thank you so much 🙇

**Pull request merged**